src/objects/zcl_abapgit_object_vcls.clas.abap

```diff
@@ -141,13 +171,52 @@ CLASS ZCL_ABAPGIT_OBJECT_VCLS IMPLEMENTATION.
 
 
   METHOD zif_abapgit_object~is_active.
-    rv_active = is_active( ).
+
+    DATA lv_changedate TYPE vcldir-changedate.
+
+    SELECT SINGLE changedate INTO lv_changedate FROM vcldir
+      WHERE vclname = ms_item-obj_name.
+
+    IF lv_changedate IS NOT INITIAL.
+* see logic in function module VIEWCLUSTER_GET_DEFINITION
+      rv_active = abap_true.
+    ELSE.
+      rv_active = abap_false.
+    ENDIF.
+
```

Comment:
Hm, how does that work? What's the connection between changedate and is_active?

Reply:
Some thing only SAP will know...
The main reason I think is that view clsuters are more a client indipendet customizing than real DDIC objects.
the logic behind this statement can be found in VIEWCLUSTER_GET_DEFINITION (line 25/26 on my release).
I also don't know a other flag to ident if a view cluster is active or not.